pandas/tests/extension/base/setitem.py

```diff
@@ -93,6 +93,90 @@ def test_setitem_iloc_scalar_multiple_homogoneous(self, data):
         df.iloc[10, 1] = data[1]
         assert df.loc[10, "B"] == data[1]
 
+    @pytest.mark.parametrize(
+        "mask",
+        [
+            np.array([True, True, True, False, False]),
+            pd.array([True, True, True, False, False], dtype="boolean"),
+        ],
+        ids=["numpy-array", "boolean-array"],
```

Comment:
numpy-boolean-array, extension-boolean-array

Reply:
The test name indicates it is about masking, so I think "numpy-array" should be sufficient